### PR TITLE
README: Update package list

### PR DESCRIPTION
**Pull request by leventebalogh**
### What changed?
Updated the structure of the main README ([view the new version &rarr;](https://github.com/grafana/plugin-tools/blob/leventebalogh/update-readme/README.md)), to make it a bit easier to scan. (This can be totally opinionated, so it's more like a conversation starter.)

|  **Before**  |    **After**    |
|--------------|-----------------|
|  Screenshot 2024-04-11 at 11 15 23  | Screenshot 2024-04-17 at 09 12 34 |

### CODE REVIEW

**Comment by jackw** [comment]
I personally prefer the table as I think it looks tidier and the buttons aren't between the name and description which is jarring to read. 

I think referring to the packages by their npm name is a nice addition as is updating the descriptions.

**Comment by leventebalogh** [comment]
Those are good inputs - just to make sure I understand correctly what you are suggesting is to go back to the table view, but use the names of the packages (e.g. `@grafana/plugin-e2e`) and also go with the updated descriptions, right? (How do you feel about getting rid of the readme column, and turning the package names into a link?)

File: README.md [hunk 1]
```diff
@@ -25,24 +25,26 @@
...
 
 ### Additional resources
 
-📖 Learn from tutorials and documentation in the [Grafana developer portal](https://grafana.com/developers).<br/>
-✨ Gain inspiration from our [plugin examples](https://github.com/grafana/grafana-plugin-examples/) to get started quickly and implement new features in your plugin.<br/>
-🛠️ Use the [Grafana plugin SDK for Go](https://github.com/grafana/grafana-plugin-sdk-go) to simplify the development of backend components.<br/>
-✅ Ensure your plugin is ready for publishing to the [Grafana plugin catalog](https://grafana.com/grafana/plugins/) with our [validator](https://github.com/grafana/plugin-validator/) tool.
+- [Grafana developer portal](https://grafana.com/developers) - learn from tutorials and read the documentation
```
[the remark this answers is not in the document]
Comment (leventebalogh) [reply README.md:44]: Haha, I 🔪d them! 🙈

**Review by mckn — COMMENTED** [review]
I like the changes in the table but I vote for keeping the emojis in the list 👍🏻

**Comment by leventebalogh** [comment]
Okie dokie, I'll revive them :)

**Review by jackw — APPROVED** [review]
LGTM (once the emojis are back)! 🚀

**Pull request merged**